src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -98,7 +98,13 @@ foam.CLASS({
     var self = this;
 
       this.addClass(this.myClass())
-      .add(this.slot(function(data, config, config$browseBorder, config$browseViews, config$browseTitle) {
+      .add(this.slot(function(data, config, config$CRUDActionsAuth$create, config$browseBorder, config$browseViews, config$browseTitle) {
+        var createAction = config$CRUDActionsAuth$create 
+            ? self.CREATE.copyFrom({
+              availablePermissions: config$CRUDActionsAuth$create
```

Comment:
Just in case we ever define stuff on that action by default

```suggestion
              availablePermissions: self.CREATE.availablePermissions.concat(config$CRUDActionsAuth$create)
```